### PR TITLE
chore: Unify testing code (ctd.) 

### PR DESCRIPTION
**Pull request by warber**
Use testify

### CODE REVIEW

File: internal/json/json_test.go [hunk 1]
```diff
@@ -19,10 +19,10 @@
 package json
 
 import (
+	"github.com/stretchr/testify/require"
+	"gotest.tools/assert"
```
Comment (jskelin) [inline internal/json/json_test.go:23]: could you please change this also

File: pkg/rest/rate_limit_test.go [hunk 2]
```diff
@@ -22,6 +22,7 @@ import (
 	"errors"
 	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/throttle"
 	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/timeutils"
+	"github.com/stretchr/testify/require"
 	"go.uber.org/mock/gomock"
 	"gotest.tools/assert"
```
Comment (jskelin) [inline pkg/rest/rate_limit_test.go:27]: could you also change this

**Review by jskelin — APPROVED** [review]

**Review by jskelin — DISMISSED** [review]

File: pkg/rest/rate_limit_test.go (hunk 2 above)
Comment (warber) [reply pkg/rest/rate_limit_test.go]: oops

File: internal/json/json_test.go (hunk 1 above)
Comment (warber) [reply internal/json/json_test.go]: oopsy

**Review by jskelin — APPROVED** [review]

**Pull request merged**